src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+body {
+  display: flex;
+  justify-content: center;
```

Comment:
```suggestion
body {
  display: flex;
  justify-content: center;
  height: 100vh;
  align-items: center;
  margin: 0;
```
it will make calendar also vertically centered and exactly in the middle of the screen
![Screenshot from 2023-08-08 15-48-00](https://github.com/mate-academy/layout_calendar/assets/33029864/267c1fde-ef1a-428b-b633-f49d7a5de75f)